pandas/tests/frame/test_reductions.py

```diff
@@ -1066,7 +1066,7 @@ def test_idxmin(self, float_frame, int_frame, skipna, axis):
         frame.iloc[15:20, -2:] = np.nan
         for df in [frame, int_frame]:
             if (not skipna or axis == 1) and df is not int_frame:
-                if axis == 1:
+                if skipna:
```

Comment:
Shouldn't we still be hitting this with the `axis == 1` case?

Reply:
When `axis==1` we run into all NAs for a single row, but when `skipna=False` we still raise the error message "Encountered an NA value with skipna=False" for performance - see the bottom half of https://github.com/pandas-dev/pandas/pull/57971#discussion_r1540125746